src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,22 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (humanYears) => {;
```

Comment:
What would happen if 0 was passed in? You have a working condition for any number greater than or equal to 1 here, so I would add a condition for zero at the start in addition to your others, then make the greater than 2 function your default:
```
  if (number === 0) {
    catYears = 0;
    dogYears = 0;
  }  else if (humanYears === 1) {
    catYears = 15;
    dogYears = 15;
  } else {
    catYears = 24 + (humanYears + 4);
    dogYears += 5 + (humanYears + 4);
  }

  return [humanYears, catYears, dogYears];
```